c7n/resources/hsm.py

```diff
@@ -110,3 +111,79 @@ class resource_type(TypeInfo):
         detail_spec = ('describe_luna_client', 'ClientArn', None, None)
         arn = id = 'ClientArn'
         name = 'Label'
+
+
+class DescribeCloudHSMBackup(DescribeSource):
+
+    def augment(self, resources):
+        for r in resources:
+            r['Tags'] = r.pop('TagList', ())
+        return resources
+
+
+@resources.register('cloudhsm-backup')
+class CloudHSMBackup(QueryResourceManager):
+
+    class resource_type(TypeInfo):
+        service = 'cloudhsmv2'
+        arn_type = 'backup'
+        permission_prefix = arn_service = 'cloudhsm'
+        enum_spec = ('describe_backups', 'Backups', None)
```

Comment:
it seems like we should have a static on the enum spec filter to only have backups in not pending delete status, thoughts?

Reply:
I think we should output all backup resources here. There is an attribute 'BackupState' in the output json so that users can further filter out resources as they see fit.

Reply:
I added additional filter to only have backups in 'READY' status. For DescribeBackups operation, you can only filter on 1 state at a time.

Reply:
`states` can take in a list as input [here](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/cloudhsmv2/client/describe_backups.html), no?

Reply:
It look like a bug on python boto3 to display backups with multiple states. I will contact their TAM to get that issue addressed. In the mean time, I reverted my PR to show all backup resources for now.